### DISCUSSION TITLE
Wanna contribute? Read description.

### DISCUSSION THREAD

**Discussion by pavittarx** [pavittarx/editorjs-html#44] [Q&A]
I am planning to work on a newer iteration of this library/parser. I am also looking for contributors for the same. Anyone up for it??

**Comment by dayuT** [comment]
您好。您的邮件我已收到，我将尽快给您回复。

**Reply by pavittarx** [reply]
Newer version is available.

**Reply by pavittarx** [reply]
this has been updated.
